src/backend/marsha/e2e/test_lti.py

```diff
@@ -698,6 +698,7 @@ def test_lti_video_upload(page: Page, live_server: LiveServer):
 
     page.click("text=Create a video")
 
+    page.wait_for_selector("text=Video creation", state="detached")
```

Comment:
in the commit description:
`s/Now we're waiting for an network idle event/Now we're waiting for a network idle event`